client/components/plans/plans-compare/index.jsx

```diff
@@ -153,17 +154,17 @@ var PlansCompare = React.createClass( {
 	comparisonTable: function() {
 		var plansColumns,
 			featuresList = this.props.features.get(),
+			hideFreePlan = this.props.hideFreePlan ? true : false,
```

Comment:
And again we probably don't need a variable for this - using `this.props.hideFreePlan` directly works fine (you're actually using both in your changes below).